README.md

```diff
@@ -62,12 +62,12 @@ $ open MSALiOS.xcworkspace
 
 This app comes pre-configured for testing. If you would like to register your own app, please follow the steps below.
 
-1. Sign in to the [Azure portal](https://portal.azure.com) using either a work or school account.
-2. In the left-hand navigation pane, select the **Azure Active Directory** blade, and then select **App registrations**.
+1. Sign in to the [Microsoft admin center](https://portal.azure.com) using either a work or school account.
+2. In the left-hand navigation pane, select the **Microsoft Entra ID** blade, and then select **App registrations**.
 3. Click on the **New registration** button at the top left of the page.
 4. On the app registration page,
    - Name your app
-   - Under **Supported account types**, select **Accounts in any organizational directory (Any Azure AD directory - Multitenant) and personal Microsoft accounts (e.g. Skype, Xbox)**
+   - Under **Supported account types**, select **Accounts in any organizational directory (Any Microsoft Entra ID directory - Multitenant) and personal Microsoft accounts (e.g. Skype, Xbox)**
```

Comment:
```suggestion
   - Under **Supported account types**, select **Accounts in any organizational directory (Any Microsoft Entra ID tenant - Multitenant) and personal Microsoft accounts (e.g. Skype, Xbox)**
```

Correcting to match the portal. 

![image](https://github.com/Azure-Samples/ms-identity-mobile-apple-swift-objc/assets/990063/6af8141f-e3fa-41bb-b92e-aa902bc95263)

Reply:
(See screenshot in my message above, the Portal says verbatim: "Entra ID tenant")

Reply:
I wasn't clear my bad. Right I see it but Microsoft right next to Entra ID tenant.  You and the portal are right in using tenant instead of directory. "Microsoft Entra ID directory" doesn't make sense and is wrong.

There is an issue that Celeste from Docs will clarify.
The issue is that the wording in the portal conflicts with the guidance. When describing the tenant, it is Microsoft Entra tenant. "ID" is not used in the product name. Here's another source https://review.learn.microsoft.com/en-us/entra/fundamentals/how-to-rename-azure-ad?branch=main#automate-bulk-editing-using-custom-code

Reply:
I wasn't clear my bad. I see it but Microsoft right next to Entra ID tenant.  You and the portal are right in using tenant instead of directory. "Microsoft Entra ID directory" doesn't make sense and is wrong. This isn't something to be fixed by the script though. The issue is in the original readme, "Azure AD Directory" was the incorrect phrasing.  Essentially "Azure Active Directory directory" so we got Microsoft Entra ID directory after the script was run because the redundant "directory" is ignored (as it should be). We can fix that manually.

The issue Celeste from Docs team will clarify is the wording in the portal. It conflicts with the guidance. When describing the tenant, it is "Microsoft Entra tenant".  There is no "ID" used in the product name. Here's another source https://review.learn.microsoft.com/en-us/entra/fundamentals/how-to-rename-azure-ad?branch=main#automate-bulk-editing-using-custom-code

Reply:
I have confirmed with Celeste that there is a mistake in the portal. The right term to describe then tenant is "Microsoft Entra Tenant". Thanks for the callout. She will be informing her team and they will be fixing this mistake.